R/render.R

```diff
@@ -4,7 +4,10 @@
 #' (knitted from R Markdown) and JavaScript code to render slides.
 #' \code{tsukuyomi()} is an alias of \code{moon_reader()}.
 #'
-#' Tsukuyomi is a genjutsu to trap the target in an illusion on eye contact.
+#' @details Tsukuyomi is a genjutsu to trap the target in an illusion on eye contact.
+#' @details If you are unfamiliar with CSS, please see the
```

Comment:
Ok, thanks :) I modified the PR. That´s the first time I am faced with a review of a PR. Can you accept it now after my changes or do I need to do something else?